<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Register</class>
 <widget class="QMainWindow" name="Register">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Register</string>
  </property>
  <property name="windowIcon">
   <iconset theme="QIcon::ThemeIcon::HelpAbout"/>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(15, 23, 36);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="inputPassword">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>220</y>
      <width>300</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>300</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>300</width>
      <height>30</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="maxLength">
     <number>100</number>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::EchoMode::Password</enum>
    </property>
    <property name="placeholderText">
     <string>Contraseña</string>
    </property>
   </widget>
   <widget class="QLabel" name="textUser">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>130</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(11, 21, 32);</string>
    </property>
    <property name="text">
     <string>Usuario:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="textTitle">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>59</y>
      <width>240</width>
      <height>61</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>240</width>
      <height>80</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 28pt &quot;Arial&quot;;
background-color: rgb(11, 21, 32);</string>
    </property>
    <property name="text">
     <string>Registrarse</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="inputUser">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>150</y>
      <width>300</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>300</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>300</width>
      <height>30</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="maxLength">
     <number>100</number>
    </property>
    <property name="placeholderText">
     <string>Usuario</string>
    </property>
   </widget>
   <widget class="QLabel" name="textPassword">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>200</y>
      <width>110</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(11, 21, 32);</string>
    </property>
    <property name="text">
     <string>Contraseña:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="textLogin">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>390</y>
      <width>220</width>
      <height>20</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>220</width>
      <height>20</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 255);
background-color: rgb(11, 21, 32);</string>
    </property>
    <property name="text">
     <string>¿Ya tienes cuenta? Inicia sesión</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="btnRegister">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>340</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>150</width>
      <height>30</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>14</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(15, 23, 36);</string>
    </property>
    <property name="text">
     <string>Registrar</string>
    </property>
   </widget>
   <widget class="QLabel" name="textConfirmPassword">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>270</y>
      <width>200</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(11, 21, 32);</string>
    </property>
    <property name="text">
     <string>Confirmar Contraseña:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="inputConfirmPassword">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>290</y>
      <width>300</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>300</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>300</width>
      <height>30</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="maxLength">
     <number>100</number>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::EchoMode::Password</enum>
    </property>
    <property name="placeholderText">
     <string>Confirmar Contraseña</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>50</y>
      <width>321</width>
      <height>371</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(11, 21, 32);
border-radius: 20px;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>label</zorder>
   <zorder>inputPassword</zorder>
   <zorder>textUser</zorder>
   <zorder>textTitle</zorder>
   <zorder>inputUser</zorder>
   <zorder>textPassword</zorder>
   <zorder>textLogin</zorder>
   <zorder>btnRegister</zorder>
   <zorder>textConfirmPassword</zorder>
   <zorder>inputConfirmPassword</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
